File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main()
{
    int height;
    // prompt the user for a positive integer input
    do
    {
        height = get_int("Enter the height of the pyramids: ");
    }
    while (height < 1);

    for (int i = 1; i <= height; i++)
    {
        // Counting the spaces
        int spaces = height - i;

        // The first pyramid
        for (int j = 0; j < spaces; j++)
        {
            printf(" ");
        }

        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // Two block gap between the pyramids
        printf("  ");

        // The second pyramid
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        printf("\n");
    }

    return 0;
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break


def build(h):
    for i in range(1, h+1):
        gap = h - i
        print(" " * gap, end="")
        print("#" * i, end="  ")
        print("#" * i)


build(height)
